misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatUtil.java

```diff
@@ -130,32 +126,55 @@ static void execute(List<String> argList, boolean helpFlag, boolean dryRun, Path
                         }
                     }
 
-                    // Check whether the given directory is not in a ballerina project.
-                    if (!FormatUtil.isBallerinaProject(sourceRootPath)) {
-                        throw LauncherUtils.createLauncherException(Messages.getNotBallerinaProject());
-                    }
-                    BLangPackage bLangPackage = FormatUtil
-                            .compileModule(sourceRootPath, getModuleName(moduleName));
+                    BuildProject project = BuildProject.load(sourceRootPath, constructBuildOptions());
```

Comment:
We need to wrap this with a try-catch to catch the ProjectException

Reply:
Done